OddOccurrencesInArray/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Reflection.Metadata.Ecma335;
+
+namespace ObjectApp
+{
+    class Solution
+    {
+        public int solution(int[] A)
+        {
+            int elementValue = A[0];
+            for (int i = 1; i < A.Length; i++)
```

Comment:
use brackets
and use spaces before if/for based on C# conventions, ask in case if you need more details